crates/dojo-world/src/manifest.rs

```diff
@@ -107,12 +91,11 @@ pub struct ComputedValueEntrypoint {
 #[serde_as]
 #[derive(Clone, Default, Debug, Serialize, Deserialize, PartialEq)]
 pub struct Contract {
-    pub name: SmolStr,
-    #[serde_as(as = "Option<UfeHex>")]
-    pub address: Option<FieldElement>,
     #[serde_as(as = "UfeHex")]
     pub class_hash: FieldElement,
-    pub abi: Option<abi::Contract>,
+    pub abi: Option<String>,
```

Comment:
All paths for ABIs will then be relative to the manifest file itself?

Reply:
yes for now, but i imagine this being an issue once we have created this struct by reading the file. we will loose the information of what is this path relative to. 

for example while using in sozo after we read this file we would need to pass around the path we have read this file to everywhere.

wdyt about instead of this being a path we can just change it to filename and make user (e.g. sozo) to be aware of the path it needs check for abi relative to `Scarb.toml` file.

cc: @tarrencev

Reply:
Hm to make sure I understand correctly, the idea is to have an `abis` base path and then this path is relative to that? I think that makes sense